socorro/processor/breakpad_transform_rules.py

```diff
@@ -248,13 +232,10 @@ class BreakpadStackwalkerRule2015(ExternalProcessRule):
     def _temp_raw_crash_json_file(self, raw_crash, crash_id):
         file_pathname = os.path.join(
             self.config.temporary_file_system_storage_path,
-            "%s.%s.TEMPORARY.json" % (
-                crash_id,
-                threading.currentThread().getName()
-            )
+            '%s.%s.TEMPORARY.json' % (crash_id, threading.currentThread().getName())
         )
         with open(file_pathname, "w") as f:
-            ujson.dump(raw_crash, f)
+            json.dump(dotdict_to_dict(raw_crash), f)
```

Comment:
This was the problematic line.

Last week, `raw_crash` would have been a Socorro DotDict. That subclasses dict, so `ujson.dump` would do the right thing with it. Then I purged Socorro DotDict from the system so then `raw_crash` here becomes a configman DotDict which does not subclass dict and thus `ujson.dump` doesn't serialize the data right.

This is exacerbated by ujson not throwing an error--it just serializes it weird:

```
app@socorro:/app$ python
Python 2.7.14 (default, Dec 12 2017, 06:55:29) 
[GCC 4.9.2] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import ujson
>>> from configman.dotdict import DotDict
>>> d = DotDict({'foo': 'bar'})
>>> ujson.dumps(d)
'["foo"]'
>>> 
```

Further, this processor rule swallows errors, so they don't show up in logs or sentry or anywhere we're monitoring. Ugh.

So the fix here is to run the `raw_crash` through `dotdict_to_dict` which gets a Python dict and that serializes just fine. On top of that, I added some logging and I purged ujson from the system because it's frustrating and problematic to have two json serialization systems in play that do different things.